src/System.Private.CoreLib/shared/System/TimeZoneInfo.cs

```diff
@@ -982,10 +982,10 @@ public bool HasSameRules(TimeZoneInfo other)
         public static TimeZoneInfo CreateCustomTimeZone(

```

Comment:
There is one more override of `CreateCustomTimeZone` missing annotations, L961 of this file.